mlflow/protos/mlflow_artifacts.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto2";
```

Comment:
Can we add a docstring to this proto file explaining which use cases it serves?

Reply:
Sure!